Atomic/syscontainers.py

```diff
@@ -1179,23 +1193,17 @@ def rollback(self, name):
         :param name: The container to rollback.
         :type name: str
         """
-        path = os.path.join(self._get_system_checkout_path(), name)
+        system_checkout_path = self._get_system_checkout_path()
+        path = os.path.join(system_checkout_path, name)
         destination = "%s.%d" % (path, (1 if os.path.realpath(path).endswith(".0") else 0))
         if not os.path.exists(destination):
             raise ValueError("Error: Cannot find a previous deployment to rollback located at %s" % destination)
 
-        installed_files = None
+        info, rpm_installed, installed_files_checksum = SystemContainers._gather_installed_files_info(system_checkout_path, name)
```

Comment:
we don't really follow this line for long lines.  I think it is fine to leave the full line.

After this, I think the patch is good to go!

Reply:
? sorry, can you explain a bit more? (me guessing) Are you suggesting the following:
```
tuple = SystemContainers._gather_installed_files_info(system_checkout_path, name)
info = tuple[0]
rpm_installed = tuple[1]
installed_files_checksum = tuple[2]
```
?

Reply:
no nevermind it was just a mistake on my side.